### PR TITLE
docs: assorted fixes for broken aspects

### LINKED ISSUE #517: [Docs] Consistent indentation in code blocks
## On which page?
https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/Navigation/NavigationOverview.html#7-displaying-content-in-a-contentcontrol

## What's wrong?
xaml have improper padding (8-sized tab, and not 4 spaces) and attibutes are misaligned
image

## Any feedback?
use space for indentation

### PR DESCRIPTION
**Pull request by lukeblevins**
GitHub Issue (If applicable): #1169 
Closes #517 

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the new behavior?

- Fix broken configuration link
- Fix broken Feed page title
- Fix broken ListFeed page title
- Fix broken code block
- Make use of spaces consistent

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/Learn/Tutorials/Authentication/HowTo-Authentication.md [hunk 1]
```diff
@@ -76,28 +76,28 @@ uid: Learn.Tutorials.Authentication.HowToAuthentication
...
-    								new RouteMap("Main", View: views.FindByViewModel<MainViewModel>()),
-    								new RouteMap("Second", View: views.FindByViewModel<SecondViewModel>(), DependsOn:"Main"),
-    				}));
+        .Register(
+            new RouteMap("", View: views.FindByViewModel<ShellViewModel>() ,
+                    Nested: new RouteMap[]
+                    {
+                                    new RouteMap("Main", View: views.FindByViewModel<MainViewModel>()),
+                                    new RouteMap("Second", View: views.FindByViewModel<SecondViewModel>(), DependsOn:"Main"),
+                    }));
```
Comment (Xiaoy312) [inline doc/Learn/Tutorials/Authentication/HowTo-Authentication.md:100]:
```suggestion
        .Register(
            new RouteMap("", View: views.FindByViewModel<ShellViewModel>() ,
                Nested: new RouteMap[]
                {
                    new RouteMap("Main", View: views.FindByViewModel<MainViewModel>()),
                    new RouteMap("Second", View: views.FindByViewModel<SecondViewModel>(), DependsOn:"Main"),
                }));
```

File: doc/Learn/Tutorials/Authentication/HowTo-Authentication.md [hunk 2]
```diff
@@ -162,15 +162,15 @@ From this walk through you can see how the IAuthenticationService can be used to
...
-    			})
+        return UnoHost
+                .CreateDefaultBuilder()
+                ...
+                .ConfigureServices((context, services) =>
+                {
+                    services
+                            .AddNativeHandler()
+                            .AddRefitClient<IDummyJsonEndpoint>(context);
+                })
```
Comment (Xiaoy312) [inline doc/Learn/Tutorials/Authentication/HowTo-Authentication.md:173]:
```suggestion
        return UnoHost
            .CreateDefaultBuilder()
            ...
            .ConfigureServices((context, services) =>
            {
                services
                    .AddNativeHandler()
                    .AddRefitClient<IDummyJsonEndpoint>(context);
            })
```

File: doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseContentControl.md [hunk 3]
```diff
@@ -12,71 +12,85 @@ uid: Learn.Tutorials.Navigation.Advanced.ContentControl
...
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-             mc:Ignorable="d">
-    <TextBlock Text="Right"
-               FontSize="24"
-               HorizontalAlignment="Center"
-               VerticalAlignment="Center" />
-</UserControl>
-```
+    ```xml
```
Comment (Xiaoy312) [inline doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseContentControl.md:15]: i think this is worse than before
i would suggestion reverting this

File: doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md [hunk 4]
```diff
@@ -19,59 +19,79 @@ uid: Learn.Tutorials.Navigation.Advanced.Panel
...
-   xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-   xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-   xmlns:local="using:UsingPanelRegion.Views"
-   xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-   xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-   mc:Ignorable="d"
-   xmlns:uen="using:Uno.Extensions.Navigation.UI"
-   xmlns:utu="using:Uno.Toolkit.UI"
-  Background="{ThemeResource MaterialBackgroundBrush}">
+    ```xml
```
Comment (Xiaoy312) [inline doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md:22]: no improvement or arguably worse

File: doc/Overview/Navigation/NavigationOverview.md [hunk 5]
```diff
@@ -204,32 +211,38 @@ The type of view to navigate to
...
-    					<TextBlock Text="Deals" />
-    				</StackPanel>
-    				<StackPanel uen:Region.Name="Profile"
-    					  Visibility="Collapsed">
-    					<TextBlock Text="Profile" />
-    				</StackPanel>
-    			</Grid>
-    		</muxc:NavigationView>
-    	</Grid>
+        <Grid>
```
Comment (Xiaoy312) [inline doc/Overview/Navigation/NavigationOverview.md:214]: no improvement or arguably worse

File: doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md (hunk 4 above)
Comment (Xiaoy312) [reply doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md:22]: but keep the tab/space change

Comment (Xiaoy312) [reply doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md:22]: > image
> -- https://github.com/unoplatform/uno.extensions/blob/dev/lubl/fix-docs/doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md#navigating-using-a-panel

😬 dont 4-space indent unless within bullet-list-item, cause that is a syntax for language-less codeblock

**Review by Xiaoy312 — COMMENTED** [review]
looks about right, beside the indented codeblock one

File: doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md (hunk 4 above)
Comment (lukeblevins) [reply doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md]: @Xiaoy312 Interesting. Didn't know that about the code block syntax. It should be addressed with the latest commit.

**Review by Xiaoy312 — APPROVED** [review]

**Pull request merged**